client/src/components/registration/RegistrationForm.js

```diff
@@ -139,6 +180,21 @@ const RegistrationForm = () => {
           </label>
           </div>
         </div>
+
+        <div className="grid-x">
+          <div className="center small-4">
+
+          <label className="welcome-message">
+            Administrator
+            <input
+              type="checkbox"
+              name="admin"
+              value={userPayload.admin}
+              onChange={onInputChange}
+            />
```

Comment:
while I appreciate the effort....this allows any random person from the internet to be an admin....probably dont want that. Instead, dont worry about adding an admin via form. Simply seeding an admin user account is fine. Lets remove this field